src/libraries/System.Security.Cryptography.Encoding/src/System/Security/Cryptography/PemFields.cs

```diff
@@ -0,0 +1,40 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace System.Security.Cryptography
+{
+    /// <summary>
+    /// Contains information about the location of PEM data.
+    /// </summary>
+    public readonly struct PemFields
+    {
+        internal PemFields(Range label, Range base64data, Range location, int decodedDataLength)
+        {
+            Location = location;
+            DecodedDataLength = decodedDataLength;
+            Base64Data = base64data;
+            Label = label;
+        }
+
+        /// <summary>
+        /// The location of the PEM, including the surrounding ecapsulation boundaries.
+        /// </summary>
+        public Range Location { get; }
+
+        /// <summary>
+        /// The location of the label.
+        /// </summary>
+        public Range Label { get; }
+
+        /// <summary>
+        /// The location of the base64 data inside of the PEM.
+        /// </summary>
+        public Range Base64Data { get; }
+
+        /// <summary>
+        /// The size of the decoded base-64, in bytes.
+        /// </summary>
+        public int DecodedDataLength { get; }
```

Comment:
I believe these properties need the `<value>` xml tag. @carlossanlop - is that correct?

Reply:
Since Visual Studio doesn't add them automatically, we consider them optional in triple slash comments, but they are highly desirable, since we will eventually have to fill them out in dotnet-api-docs when these comments get ported there.

Reply:
> since we will eventually have to fill them out in dotnet-api-docs when these comments get ported there

So... because we have to eventually add them anyway, I think we should get in the habit of adding them in triple slash comments. This helps ensuring the docs/work happens as close to the PR/original API author. I don't see a benefit of leaving them out.

Reply:
For properties, long term, we probably want to either provide only value or only summary, and let the tool calculate the one from the other.  (Value -> summary is the easiest; but "just summary" is the most natural).

If a property summary starts with "Gets or sets " or "Gets ", value is almost always `summary.AsSpan(prefix, 1).ToUpperString() + summary.AsSpan(prefix+1)`.  If that turns out to not be the case (too many people correcting things in api-docs PRs) then we can do something else.

But given that it's almost redundant information (in English), I'm all for letting a tool do it, not a human.